### ISSUE TITLE
image currently in view is moved 50% down while lazy loading

### ISSUE THREAD

**Issue by gangelop** [Psycojoker/prosopopee#122]
It looks like this:
![2020-09-20-1600557358_1920x1080](https://user-images.githubusercontent.com/1893345/93690966-8ab92400-fae7-11ea-92d2-7452d1250c2e.png)
Once lazy loading is finished, the image returns in full view. When you move to the next image it does the same while the others are being loaded.

I hate this bug because:
* I don't want to show my site to anybody in this state. It's jarring and goes against the otherwise nice and clean design.
* I have no experience with frontend and I don't even know where to begin investigating this, especially due to it's elusive nature. All I can think of right now is git bisect...

Anybody out there pls help! I was able to fix all the other bugs that annoyed me (see pull requests). But it would be nice to have help with this one.

**Comment by gangelop** [comment]
This issue is resolved by updating to https://github.com/feimosi/baguetteBox.js/releases/tag/v1.11.1

**Comment by halloleo** [comment]
Thanks for the tip in the follow up, @gangelop! I just start using prosopopee and this bug annoyed me too. The new baguetteBox fixes it for me as well.

**Comment by gangelop** [comment]
@halloleo I'm so happy this is solved! \o/

@Psycojoker I'm not sending a PR for this one because it would be the same amount of work for you to compare the minified js to my PR for review compared to doing it yourself. So this is a gentle reminder for an easy fix of updating `baguetteBox` to `v.11.1` to fix this issue.

Thanks for reviewing and merging all my other PRs. Have a nice week.

**Comment by Psycojoker** [comment]
@gangelop that's done and I've release a 1.1.1 with it :) (and a 1.1 yesterday)

**Comment by Psycojoker** [comment]
Closing therefor, don't hesitate to reopen if needed

**Comment by halloleo** [comment]
Hi have just tried the newest release 1.1.1 from pypi, but it does not seem to incoroprate a new baguetteBox.js:

* `baguetteBox.min.js` in Prosopopee has version 1.11.0
* `baguetteBox.min.js` from https://github.com/feimosi/baguetteBox.js has version 1.11.1

The `baguetteBox.min.css` files are different too.

**Comment by Psycojoker** [comment]
That's really weird because this is not what the commit is saying:

![image](https://user-images.githubusercontent.com/41827/95860899-fa29d880-0d60-11eb-801c-3efb0bac6498.png)

I might have made a mistake using twine when uploading the package (second time I'm using it)

**Comment by Psycojoker** [comment]
So, turns out, setup.py has bad caching for wheel generation in the "build/" folder which doesn't detect file differences for some reason (but only for the wheel, the tar.gz archive is correct), I've made a new release with the correct files, thx for repporting this bug :)

**Comment by halloleo** [comment]
I have installed the latest release fro pypi (prosopopee 1.1.2) and baguetteBox works now correctly.

Thanks heaps for fixing this, @Psycojoker!
